scripts/install-packages.sh

```diff
@@ -4,7 +4,7 @@
 # install packages
 ################################################################################
 
-PKGS_TO_INSTALL="${CONF_PKGS_TO_INSTALL}"
+PKGS_TO_INSTALL=(`echo "${CONF_PKGS_TO_INSTALL}"`)
```

Comment:
This commit was unnecessary :sweat_smile:

Reply:
Converts a string from the config file into a list. It's needed else pacstrap receives a single argument containing all the packages (which fails) as opposed to a list of packages.

Reply:
Shouldn't `${CONF_PKGS_TO_INSTALL[@]}` be more appropriate in that case?